templates/default.md

```diff
@@ -79,12 +79,12 @@
 
 ## 🤝 Contributing
 
-Contributions, issues and feature requests are welcome !<br />Feel free to check [issues page](<%= contributingUrl %>).
+Contributions, issues and feature requests are welcome!<br />Feel free to check [issues page](<%= contributingUrl %>).
 <% } -%>
 
 ## Show your support
 
-Give a ⭐️ if this project helped you !
+Give a ⭐️ if this project helped you!
```

Comment:
Or you sync with master ? Those are supposed to be merged already 😅

Reply:
Started the work on this before the other PR was merged 😅I'll sync.